examples/chunked.rs

```diff
@@ -1,14 +1,10 @@
 use async_std::task;
-use tide::{Body, Response, StatusCode};
 
 fn main() -> Result<(), std::io::Error> {
     task::block_on(async {
         let mut app = tide::new();
-        app.at("/").get(|_| async {
-            let mut res = Response::new(StatusCode::Ok);
-            res.set_body(Body::from_file(file!()).await.unwrap());
-            Ok(res)
-        });
+        app.at("/")
+            .get(|_| async { Ok(tide::Body::from_file(file!()).await?) });
```

Comment:
As an aside: would be fun if we could add a bound where we have `From<Path> for Response` that turns this into a shorthand. -- Wouldn't quite work in this case since it's a `&'static str`, but probably in some other cases it would.

Reply:
I looked into this as a distinct PR, but it requires an awkward task::block_on because From::from isn't async so I'm not convinced it's worthwhile